Frontend/src/Components/Motor_Motion/Motor_Motion.js

```diff
@@ -0,0 +1,445 @@
+                        <Text fontSize='xs'>Accelerator</Text>
+                        <Spacer/>
+                        <Text fontSize='xs'>{props.data?.accelerator[0].toFixed(0) ?? -1.0} {CONSTANTS.accelerator.UNIT}</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={2}
+                    colStart={2}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                >
+                    <Center direction='row' pl='1' pr='1' h='100%'>
+                        <Text fontSize='xs'>Accelerator Out</Text>
+                        <Spacer/>
+                        <Text fontSize='xs'>{props.data?.accelerator_out[0].toFixed(1) ?? -1.0} {CONSTANTS.accelerator_out.UNIT}</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={3}
+                    colStart={2}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                >
+                    <Center direction='row' pl='1' pr='1' h='100%'>
+                        <Text fontSize='xs'>Speed</Text>
+                        <Spacer/>
+                        <Text fontSize='xs'>{props.data?.speed[0].toFixed(0) ?? -1.0} {CONSTANTS.speed.UNIT}</Text>
+                    </Center>
+                </GridItem>
+                <GridItem
+                    rowStart={4}
+                    colStart={2}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                    pl='1' 
+                    pr='1'
+                    pb='1'
+                >
+                    <RangeCell
+                        fontSize='xs'
+                        label='Motor Power'
+                        data={props.data?.motor_power[0] ?? -1.0}
+                        digits={0}
+                        unit={CONSTANTS.motor_power.UNIT}
+                        min={CONSTANTS.motor_power.MIN}
+                        max={CONSTANTS.motor_power.MAX}
+                    />
+                </GridItem>
+                <GridItem
+                    rowStart={5}
+                    colStart={2}
+                    borderTop='1px'
+                    borderLeft='1px'
+                    borderRight='1px'
+                    borderBottom='1px'
+                    pl='1' 
+                    pr='1'
+                    pb='1'
+                >
+                    <RangeCell
+                        fontSize='xs'
+                        label='Motor Current'
+                        data={props.data?.motor_current[0] ?? -1.0}
+                        digits={0}
+                        unit={CONSTANTS.motor_current.UNIT}
+                        min={CONSTANTS.motor_current.MIN}
+                        max={CONSTANTS.motor_current.MAX}
+                    />
```

Comment:
```suggestion
                    <RangeCell
                        fontSize='xs'
                        rangeBarH='1vh'
                        label='Motor Current'
                        data={props.data?.motor_current[0] ?? -1.0}
                        digits={0}
                        unit={CONSTANTS.motor_current.UNIT}
                        min={CONSTANTS.motor_current.MIN}
                        max={CONSTANTS.motor_current.MAX}
                    />
```
Again, at least on my end, decreasing the RangeBar height to 1vh fits better. You could add a `rangeBarH` prop to the `RangeBar` in `RangeCell.js`.